lib/widgets/consult_pad.dart

```diff
@@ -232,12 +288,31 @@ class _ConsultPadWidgetState extends State<ConsultPadWidget> {
       ),
     );
   }
+  SlidableAction _removeInvestigationAction(OmrsOrder investigation){
+    return SlidableAction(
+        onPressed: (context){
+          setState(() {
+            Provider.of<ConsultationBoard>(context, listen: false)
+                .removeInvestigation(investigation);
+          });
+        },
+      borderRadius: BorderRadius.all(Radius.circular(100)),
+      autoClose: true,
+        icon: Icons.delete,
+        backgroundColor: Color.fromRGBO(240, 39, 22, 0.6),
+      foregroundColor: Colors.white,
+    );
+  }
 
   SlidableAction _removeConditionAction(ConditionModel condition) {
     return SlidableAction(
+
       onPressed: (context) {
-        Provider.of<ConsultationBoard>(context, listen: false)
-            .removeCondition(condition);
+        setState(() {
```

Comment:
same comment as above. the consult pad sections are rendered within a consumer of consultationBoard